code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -69,24 +73,38 @@ GLOBAL_LIST_INIT(sandbag_recipes, list ( \
 	. = ..()
 	. += GLOB.sandbag_recipes
 
-/obj/item/emptysandbag
-	name = "empty sandbag"
+/obj/item/stack/sheet/mineral/emptysandbags
```

Comment:
why make empty sandbags a mineral?

Reply:
There really is no reason other than to keep them similar to the filled sandbags which are also under the mineral path... I am sure there was some reason I did that originally but I don't remember why.